dev-tools/ecs-migration.yml

```diff
@@ -6,44 +6,46 @@
 #
 # - from: source-field-in-6.x
 #   to: target-filed-in-ECS
-#   # Alias field is useful for fields where there is a 1-1 mapping from old to new
-#   alias: true-if-alias-is-required-in-6x
+#   # Alias field is useful for fields where many-11 mapping from new to old are needed
```

Comment:
many-1

Reply:
done

Reply:
I don't see that fix. Have you pushed? 😂